vm/src/builtins/dict.rs

```diff
@@ -989,6 +989,11 @@ impl PyDictKeys {
     fn contains(zelf: PyRef<Self>, key: PyObjectRef, vm: &VirtualMachine) -> PyResult<bool> {
         zelf.dict().contains(key, vm)
     }
+
+    #[pymethod(magic)]
+    fn sizeof(zelf: PyRef<Self>) -> usize {
+        zelf.dict().sizeof()
```

Comment:
`dict` has data but `dict_keys` is just a view of dict. So it would be not the same

Reply:
I think this could just be `size_of::<Self>` and could be implemented directly on `DictView`.